packages/lib/hooks/useLocale.ts

```diff
@@ -1,7 +1,7 @@
 import { useTranslation } from "next-i18next";
 
-export const useLocale = () => {
-  const { i18n, t } = useTranslation("common");
+export const useLocale = (namespace: Parameters<typeof useTranslation>[0] = "common") => {
```

Comment:
So we can pass more [namespaces](https://github.com/isaachinman/next-i18next#loading-namespaces-dynamically-client-side)